src/ompl/geometric/planners/fmt/src/FMT.cpp

```diff
@@ -478,10 +581,12 @@ bool ompl::geometric::FMT::expandTreeFromNode(Motion **z)
                 x->setParent(yMin);
                 x->setCost(cMin);
                 x->setHeuristicCost(opt_->motionCostHeuristic(x->getState(), goalState_));
+                yMin->getChildren().push_back(x);
+
                 // Add x to Open
-                Open_new.push_back(x);
-                // Remove x from Unvisited
-                x->setSetType(Motion::SET_CLOSED);
+                Open_.insert(x);
```

Comment:
Yes, I remember that as well. But I couldn't remember what we chose. There is no real difference in time (actually I still think that it does not make any difference in the theory as well hehe). But I can leave it as it was, to avoid unnecessary changes. Another plus is that this way is closer to the paper. However in the [BFMT PR ](https://github.com/ompl/ompl/pull/70) we still use the `Open_new` set so for consistency I think  I will remove these changes.